linode/manager
# refactor: [M3-6302] - MUIv5 Migration - Components > CircleProgress

## Description 📝
- Migrate CircleProgress component from JSS to styled components (Emotion)
- Use named export instead of default

## How to test 🧪
- Check places in the app where the CircleProgress component is used
- Ensure that the CircleProgress story renders in Storybook
